DataFormats/MuonReco/interface/Muon.h

```diff
@@ -219,7 +219,11 @@ namespace reco {
       MultiIsoMedium         = 1UL<<26,   // miniIso with ptRatio and ptRel 
       PuppiIsoLoose          = 1UL<<27,
       PuppiIsoMedium         = 1UL<<28,
-      PuppiIsoTight          = 1UL<<29
+      PuppiIsoTight          = 1UL<<29, 
+      MvaVTight              = 1UL<<30,   // > 0.45
+      MvaVVTight             = 1UL<<31,   // > 0.9
+      LowPtMvaLoose          = 1UL<<32,   // > -0.6
+      LowPtMvaMedium         = 1UL<<33,   // > -0.2
```

Comment:
I think that there is little value to specify the values of the cuts in the header of reco::Muon. These are not fundamental constants and will likely become obsolete in the next update.